internal/model/modeldecoder/nullable/nullable.go

```diff
@@ -77,12 +78,23 @@ func init() {
 		switch iter.WhatIsNext() {
 		case jsoniter.NilValue:
 			iter.ReadNil()
-		default:
+		case jsoniter.NumberValue:
 			us := iter.ReadInt64()
 			s := us / 1000000
 			ns := (us - (s * 1000000)) * 1000
 			(*((*TimeMicrosUnix)(ptr))).Val = time.Unix(s, ns).UTC()
 			(*((*TimeMicrosUnix)(ptr))).isSet = true
+		case jsoniter.StringValue:
+			tstr := iter.ReadString()
+			t, err := time.Parse("2006-01-02T15:04:05.999-0700", tstr)
```

Comment:
Can we use `time.RFC3339` here? In my brief test, it accepts the fractional (milliseconds) component.

Reply:
It fails to parse the timezone correctly, [example](https://go.dev/play/p/2W16kCESqyT)

Reply:
I see, thanks!